### PR TITLE
KAA-1617: MariaDB cluster setup - reviewed. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md [hunk 1]
```diff
@@ -10,266 +10,241 @@ sort_idx: 30
...
-To fix this, copy the contents of the maintenance file to each individual node as follows:
+   ```bash
+   [mysqld]
+   wsrep_provider=/usr/lib/galera/libgalera_smm.so
+   wsrep_provider_options="gcache.size=256M; gcache.page_size=128M"
+   wsrep_cluster_address=gcomm://172.1.1.1,172.2.2.2,172.3.3.3
+   wsrep_cluster_name="MariaDB_Cluster"
+   wsrep_node_address="ubuntu-node1"
+   wsrep_node_name="ubuntu-node1"
+   wsrep_sst_auth="root:'your password'"
```
Comment (v-chyzhevskyi) [inline doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md]: please remove line
wsrep_sst_auth="root:'your password'"

File: doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md [hunk 2]
```diff
@@ -10,266 +10,241 @@ sort_idx: 30
...
    ```bash
-     sudo nano /etc/mysql/debian.cnf
+   [mysqld]
+   wsrep_provider=/usr/lib/galera/libgalera_smm.so
+   wsrep_provider_options="gcache.size=256M; gcache.page_size=128M"
+   wsrep_cluster_address=gcomm://172.1.1.1,172.2.2.2,172.3.3.3
+   wsrep_cluster_name="MariaDB_Cluster"
+   wsrep_node_address="ubuntu-node2"
+   wsrep_node_name="ubuntu-node2"
+   wsrep_sst_auth="root:'your password'"
```
Comment (v-chyzhevskyi) [inline doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md]: please remove line
wsrep_sst_auth="root:'your password'"

File: doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md [hunk 3]
```diff
@@ -10,266 +10,241 @@ sort_idx: 30
...
-     socket   = /var/run/mysqld/mysqld.sock
-     basedir  = /usr
+   [mysqld]
+   wsrep_provider=/usr/lib/galera/libgalera_smm.so
+   wsrep_provider_options="gcache.size=256M; gcache.page_size=128M"
+   wsrep_cluster_address=gcomm://172.1.1.1,172.2.2.2,172.3.3.3
+   wsrep_cluster_name="MariaDB_Cluster"
+   wsrep_node_address="ubuntu-node3"
+   wsrep_node_name="ubuntu-node3"
+   wsrep_sst_auth="root:'your password'"
```
Comment (v-chyzhevskyi) [inline doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md]: please remove line
wsrep_sst_auth="root:'your password'"

File: doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md]: Done.

File: doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md]: Done.

File: doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md]: Done.

**Review by v-chyzhevskyi — APPROVED** [review]

**Review by vadimol — APPROVED** [review]

File: doc/Administration-guide/System-installation/Cluster-setup/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
----
+﻿---
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Cluster-setup/index.md:1]: There is a non-printable symbol at the beginning of this file. It causes the problem described in [KAA-1668](http://jira.kaaproject.org/browse/KAA-1668)

Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/index.md:1]: This was fixed in other subsequent PR (don't remember which one).

**Pull request merged**